Documentation/Signoffs/Shifter.md

```diff
@@ -0,0 +1,50 @@
+Gear Shifter Subsystem
+--------------------------------------
+### Function of the Subsystem
+The gear shifter subsystem will capture the OEM signal from the gear shifter sensor by splicing into the wiring and storing the signal on a local microcontroller. 
+### Specifications & Constraints
+-	Must use OEM gear shifter sensor
+-	Must not have any exposed or loose wires according to NESC standard
+-	Shall transmit the gear shifter data at a refresh rate no slower than 100 ms
+-	Shall have the same accuracy as the OEM gear shifter sensor
+-	Shall store the signal on a local microcontroller
+
+### Wiring Schematic
+
+![image](https://user-images.githubusercontent.com/117474540/214940419-ab1ae883-7997-4695-af02-b0b6908a175d.png)
+
+###### _Figure 1: Splicing OEM wiring to a local microcontroller
+
+### Analysis
```

Comment:
Much of this is not analysis. Making claims is not the same as showing that something is true.

The minimum rate of signal sampling hasn't been established and you haven't shown that the rate is met.

The circuit protection article you cited is not an analysis of this application.